flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/dynamodbstreams/FlinkDynamodbStreamsConsumer.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.connectors.dynamodbstreams;
```

Comment:
All of the newly added packages should be under `org.apache.flink.streaming.connectors.kinesis` as they are part of the `flink-connector-kinesis` module.

Reply:
These are just packages with the module `flink-connector-kinesis` and the package prefix should reflect that.

Reply:
In that case I don't mind putting the files into a separate directory, e.g., `flink-connector-dynamodbstreams`.  WDYT ?

Reply:
I don't think that will work: We had already determined in our internal work that we need a single shaded AWS dependency. I don't think it's desirable either, because it is just a slight variation of the Kinesis consumer that does not deserve a separate module.